tools/individual-pkg-gen/copy-files-removing-deps.ts

```diff
@@ -150,10 +166,19 @@ function copyFilesRemovingDependencies(): void {
         // (as the unstable packages don't use direct dependencies,
         // but instead a combination of devDependencies + peerDependencies)
         fileProcessed = true;
-        const esLintRcContents = fs.readFileSync(source);
-        fs.outputFileSync(destination, Buffer.concat([esLintRcContents,
-          Buffer.from("\nbaseConfig.rules['import/no-extraneous-dependencies'] = ['error', " +
-              '{ devDependencies: true, peerDependencies: true } ];\n')]));
+        const esLintRcLines = fs.readFileSync(source).toString().split('\n');
+        const resultFileLines = [];
+        for (const line of esLintRcLines) {
+          resultFileLines.push(line);
+          // put our new line right after the parserOptions.project setting line,
+          // as some files export a copy of this object,
+          // in which case putting it at the end doesn't work
+          if (line.startsWith('baseConfig.parserOptions.project')) {
+            resultFileLines.push("\nbaseConfig.rules['import/no-extraneous-dependencies'] = ['error', " +
+              '{ devDependencies: true, peerDependencies: true } ];\n');
+          }
+        }
```

Comment:
Should we change this directly on the v2 branch here - https://github.com/aws/aws-cdk/blob/master/tools/cdk-build-tools/config/eslintrc.js, and drop all this extra logic ?

Reply:
That would change the rules for *all* packages in v2, not just the alpha packages, correct? These alpha packages have slightly different dependency setups, which requires the customization to the rule.

Reply:
Ok, I'm fine with this.